app/assets/stylesheets/partials/anime-page.css.scss

```diff
@@ -701,4 +701,3 @@
     line-height: 38px;
   }
 }
-
```

Comment:
Well I use Atom and as far as I know it adds an empty line at the end of the file by default.
What really confuses me is that as far as I understand it this diff says that the line was removed but it is there for me.

Anyway I'll just checkout another branch take the unchanged file from there and replace it. Don't know why I didn't think of that in the first place.